packages/dd-trace/src/plugins/util/ci.js

```diff
@@ -277,6 +289,16 @@ module.exports = {
           GITHUB_RUN_ATTEMPT
         })
       }
+      if (GITHUB_BASE_REF) { // `pull_request` or `pull_request_target` event
+        tags[GIT_PULL_REQUEST_BASE_BRANCH] = GITHUB_BASE_REF
+        try {
+          const eventContent = getGitHubEventPayload()
+          tags[GIT_PULL_REQUEST_BASE_BRANCH_SHA] = eventContent.pull_request.base.sha
```

Comment:
the `getGitHubEventPayload` method is doing `return` when `GITHUB_EVENT_PATH` is not set. 
You are not checking here that `eventContent` has value, so this could crash if `GITHUB_EVENT_PATH` is not set.

Reply:
and, could happen that the ci is running on master or a branch without PR? In that case, we could have the file in `env.GITHUB_EVENT_PATH`, but maybe without `pull_request` property.

Reply:
ups, you can forget these messages, I haven't see the try catch ere 🤦